### PR TITLE
:bug: (autocomplete) height of the input box

### PR DESCRIPTION
**Pull request by MatissJanis**
Fix for: https://github.com/actualbudget/actual/issues/773#issuecomment-1499235857

### CODE REVIEW

File: packages/desktop-client/src/components/accounts/Filters.js [hunk 1]
```diff
@@ -536,7 +536,7 @@ function normalizeValue(value, isMulti) {
     if (Array.isArray(value)) {
       return value;
     }
-    return value.split(', ');
+    return String(value).split(', ');
```
[the remark this answers is not in the document]
Comment (j-f1) [reply packages/desktop-client/src/components/accounts/Filters.js:539]: `String(["a", "b", "c"]) === "a,b,c"` fyi

**Comment by j-f1** [comment]
Now the padding is wrong again :( (and the height is now 1px shorter for me in Safari than the regular input)

Screenshot_2023-04-06 13 23 50

Maybe increase the height of the regular input instead?

File: packages/desktop-client/src/components/accounts/Filters.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MatissJanis) [reply packages/desktop-client/src/components/accounts/Filters.js]: Good idea

File: packages/desktop-client/src/components/accounts/Filters.js [hunk 2]
```diff
@@ -244,7 +244,7 @@ function ConfigureField({
               field={field}
               subfield={subfield}
               type={type === 'id' && op === 'contains' ? 'string' : type}
-              value={normalizeValue(value, op === 'oneOf')}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/accounts/Filters.js:247]: I reverted this PR basically https://github.com/actualbudget/actual/pull/855

The new implementation actually exposes a bunch of other issues that are not so simple to address. So I'll revert and re-open the issue. And hopefully get to fixing it sometime later.

**Comment by MatissJanis** [comment]
The spacing situation is now improved. Not absolutely pixel-perfect (like 1px issue). But IMO good enough.

**Comment by j-f1** [comment]
> I want to finally stop spending my time on pushing 1/2 pixels around and get to implementing things that actually matter.

💯

**Review by j-f1 — APPROVED** [review]

**Pull request merged**